cmd/der2ascii/writer.go

```diff
@@ -355,6 +355,40 @@ func derToASCIIImpl(out *bytes.Buffer, in []byte, indent int, stopAtEOC bool) []
 					// Emit the remaining as a DER element.
 					derToASCIIImpl(out, elem.body[1:], indent+1, false) // Adds a trailing newline.
 					addLine(out, indent, "}")
+				} else if len(elem.body) >= 1 && len(elem.body) <= 5 && elem.body[0] < 8 {
```

Comment:
I believe this will incorrectly encode {0x07} as b\`\` and break round-tripping.